<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CT_gui</class>
 <widget class="QMainWindow" name="CT_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>521</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>521</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>521</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CombAlva Tracked</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/icon/CT_icon.ico</normaloff>:/icon/CT_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>621</width>
     <height>478</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>621</width>
     <height>478</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>625</width>
      <height>482</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Character Info</string>
     </attribute>
     <widget class="QTableWidget" name="CharDataTable">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>330</width>
        <height>420</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>49</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <column>
       <property name="text">
        <string>Character</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Max HP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>HP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>AC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Init.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Condition</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="CharEdit">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>30</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="MaxHPEdit">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>50</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="ACEdit">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>90</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="InitiativeEdit">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>110</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="AddCharButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Character</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UndoCharButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Character</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StartCombButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>200</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Combat</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="CombatCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>30</width>
        <height>23</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Char">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Character</string>
      </property>
     </widget>
     <widget class="QLabel" name="MaxHP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Max HP</string>
      </property>
     </widget>
     <widget class="QLabel" name="AC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QLabel" name="Initiative">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Initiative</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cond">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Condition</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CondComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>130</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Normal</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLength</enum>
      </property>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prone</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hiden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grappled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Charmed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Frightened</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poisoned</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blind</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Incapacited</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paralised</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Petrified</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restrained</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stunned</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unconscious</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Invisible</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Deafened</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stable</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Revived</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="CurrentHP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current HP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CurrentHPEdit">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>70</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Rounds</string>
     </attribute>
     <widget class="QTableWidget" name="CharOrderTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>330</width>
        <height>420</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>49</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <column>
       <property name="text">
        <string>Character</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>HP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>AC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Init.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Condition</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="DmgDealtEdit">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>60</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Damage">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Damage</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DmgDealt">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Damage dealt</string>
      </property>
     </widget>
     <widget class="QComboBox" name="DmgTargetComboBox">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>80</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DmgTarget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Damage target</string>
      </property>
     </widget>
     <widget class="QLabel" name="CondDealt">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Condition dealt</string>
      </property>
     </widget>
     <widget class="QLabel" name="CondTarget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Condition target</string>
      </property>
     </widget>
     <widget class="QLabel" name="Condition">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Condition</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="CondTargetComboBox">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>180</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DmgHealed">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>260</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Damage healed</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DmgHealedEdit">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>260</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="HealTarget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Healing target</string>
      </property>
     </widget>
     <widget class="QLabel" name="Heal">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Heal</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="DmgHealedComboBox">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>280</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Round">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>420</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Round</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextTurnButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Turn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DealActionButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>330</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Deal Action</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CondDealtComboBox">
      <property name="geometry">
       <rect>
        <x>471</x>
        <y>160</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prone</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hiden</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grappled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Charmed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Frightened</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poisoned</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blind</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Incapacited</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paralised</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Petrified</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restrained</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stunned</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unconscious</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Invisible</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Deafened</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stable</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Revived</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="RoundNumber">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="EndCombatButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>End Combat</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionDice"/>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDice">
   <property name="text">
    <string>Dice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
